libs/sdk-ui-dashboard/src/drill/hooks/useDrillToAttributeUrl.ts

```diff
@@ -0,0 +1,26 @@
+// (C) 2020-2021 GoodData Corporation
+
+import { useDashboardCommandProcessing } from "../../dashboard/useDashboardCommandProcessing";
+import {
+    DashboardDrillToAttributeUrlTriggered,
+    DashboardCommandFailed,
+    drillToAttributeUrl,
+} from "../../model";
+
+/**
+ * @internal
+ */
+export const useDrillToAttributeUrl = (
+    onSuccess?: (event: DashboardDrillToAttributeUrlTriggered) => void,
```

Comment:
We should probably make this a config object, like in other hooks. This would allow us to specify any combination of the callbacks.